src/utils/sentry/setup.ts

```diff
@@ -0,0 +1,12 @@
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
+
+Sentry.init({
+  dsn: 'https://c170b1643b064f2cb57b2204e1e3bf5f@o4504239616294912.ingest.sentry.io/4504247590060032',
```

Comment:
```DSNs are safe to keep public because they only allow submission of new events and related event data; they do not allow read access to any information.```

https://docs.sentry.io/product/sentry-basics/dsn-explainer/